readme.md

```diff
@@ -0,0 +1,728 @@
+# Online Book Store
+
+---
+
+## Introduction
+
+The Online Book Store project was inspired by the need for an efficient online platform for book enthusiasts to explore,
+shop for, and manage their book collections. The aim is to provide a user-friendly interface for shoppers and robust
+tools for managers to organize and monitor the store.
+
+## Technologies and Tools
+
+- **Spring Boot:** A Java-based framework for building web applications.
+- **Spring Security:** Provides authentication and authorization capabilities.
+- **Spring Data JPA:** Simplifies database interactions within Hibernate.
+- **Swagger:** Allows for API documentation and testing.
+- **MySQL:** Used as the relational database to store book and user data.
+- **Liquibase:** Library for tracking, managing, and applying database schema changes.
+- **Docker:** A platform for delivering software in containers.
+
+## Video Presentation
+
+[![https://youtu.be/je94b48P8GA?si=BjrNyUUWF0zb-knZ](https://img.youtube.com/vi/je94b48P8GA/0.jpg)](https://youtu.be/je94b48P8GA?si=BjrNyUUWF0zb-knZ)
+
+## Functionalities
+
+### Shoppers (User)
+
+#### Authentication:
+
+The API uses token-based authentication. You must obtain a JSON Web Token (JWT) and include it in the Authorization
+header for protected routes.
+
+- **Login**:
```

Comment:
OPTIONAL: It makes sence to add postman collection examples separately, and in README only mention what controllers can do